### PR TITLE
Card topics update

### PR DESCRIPTION
**Pull request by StefanIvanov**
Updating cards topic and adding a new one for the skeleton cards. Since there will be no custom cards, for now, that topic is not present anymore.

### CODE REVIEW

**Comment by StefanIvanov** [comment]
Since the card is not in the master of abstract, use the NEXT branch to review the topic.

File: en/components/card.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+<img class="responsive-img" src="../images/card_media_nav.png" srcset="../images/card_media_nav@2x.png 2x" />
+
+<img class="responsive-img" src="../images/card_media_image.png" srcset="../images/card_media_image@2x.png 2x" />
+
+<img class="responsive-img" src="../images/card_body.png" srcset="../images/card_body@2x.png 2x" />
+
+## Header
+
+The Card header, through the use of `Smart Layout` in Sketch, supports various layouts including an Avatar Thumbnail, Title and Subtitle elements.
```
Comment (YoannaIvanova) [inline en/components/card.md:40]:
```suggestion
The Card header, through the use of `Smart Layout` in Sketch, supports various layouts including an Avatar Thumbnail, Title, and Subtitle elements.
```

File: en/components/card-skeleton.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+title: Skeleton Card - Design System Component
+_description: The Skeleton Cards is a representation of a Card component that is shown while data is being loaded in the background to provide content for a collection of cards.
```
Comment (YoannaIvanova) [inline en/components/card-skeleton.md:3]:
```suggestion
_description: The Skeleton Card is a representation of a Card component that is shown while data is being loaded in the background to provide content for a collection of cards.
```

File: en/components/card-skeleton.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+# Skeleton Card
+
+Use the Skeleton Card Component to layout the same type of information that you would in the standard one, but to also indicate an application state when no data is present at the moment e.g. when data is loaded from a remote datasource and there is either lack of connectivity, or it takes some time. The Skeleton Card is just a visual representation of a [Card](card.md) with known layout, but without any data to show.
```
Comment (YoannaIvanova) [inline en/components/card-skeleton.md:9]:
```suggestion
Use the Skeleton Card Component to layout the same type of information that you would in the standard one, but to also indicate an application state when no data is present at the moment e.g. when data is loaded from a remote data source and there is either lack of connectivity, or it takes some time. The Skeleton Card is just a visual representation of a [Card](card.md) with a known layout, but without any data to show.
```

File: en/components/card-skeleton.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+<img class="responsive-img" src="../images/card_skeleton_demo.png" srcset="../images/card_skeleton_demo@2x.png 2x" />
+
+## Areas
+
+The Skeleton Card supports the same four areas like the standard [Card](card.md): `image`, `header`, `body`, and `actions`. 
+
+## Image and Body
+
+The skeleton image and body areas support overrides either for an Array of shortcuts, an Image, which also covers the the maps available in the standard [Card](card.md), or a Paragraph.
```
Comment (YoannaIvanova) [inline en/components/card-skeleton.md:20]:
```suggestion
The skeleton image and body areas support overrides either for an Array of shortcuts, an Image, which also covers the maps available in the standard [Card](card.md), or a Paragraph.
```

File: en/components/card.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+|         |                                                                                                |
+| ------- | ---------------------------------------------------------------------------------------------- |
+| image   | <img class="responsive-img" src="../images/card_media_image.png" srcset="../images/card_media_image@2x.png 2x" /> |
+| header  | <img class="responsive-img" src="../images/card_header.png" srcset="../images/card_header@2x.png 2x" /> |
+| body    | <img class="responsive-img" src="../images/card_content_image.png" srcset="../images/card_body@2x.png 2x" /> |
+| actions | <img class="responsive-img" src="../images/card_actions_buttons_icons.png" srcset="../images/card_actions_buttons_icons@2x.png 2x" /> |
+
+## Image and Body
+
+The image and body Card areas support the same collection of eleven overrides: 6 continent maps for the populated world areas, one World Map, and one Navigation Map, as well as Array of shortcuts like contacts, Image, and Paragraph for a short description text.
```
Comment (YoannaIvanova) [inline en/components/card.md:28]:
```suggestion
The image and body Card areas support the same collection of eleven overrides: 6 continent maps for the populated world areas, one World Map, and one Navigation Map, as well as an Array of shortcuts like contacts, Image, and Paragraph for a short description text.
```

File: en/components/card-skeleton.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+## Skeleton Card Layout
+
+Through the use of `Smart Layout` in Sketch, the Skeleton Card can produce various layouts by excluding certain areas or elements within one area or another to create a myriad of distinct variants starting with the same Component initially and consistent with the standard [Card](card.md).
+
+<img class="responsive-img" src="../images/card_skeleton_demo.png" srcset="../images/card_skeleton_demo@2x.png 2x" />
+
+## Usage
+
+When creating layouts with multiple cards, avoind combining standard and skeleton cards. Data is either present for all cards, or for none of them, thus the two types should not be used within the same grid layout.
```
Comment (YoannaIvanova) [inline en/components/card-skeleton.md:56]:
```suggestion
When creating layouts with multiple cards, avoid combining standard and skeleton cards. Data is either present for all cards, or for none of them, thus the two types should not be used within the same grid layout.
```

**Review by YoannaIvanova — APPROVED** [review]

File: en/components/card.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+Use the Card Component to display information for a single object through an image, map, or other media content, as well as text, allowing relevant quick actions to be triggered such as sharing, liking, bookmarking, etc. It usually works as an overview and entry point for more detailed information. Cards with various content and layout can be combined to create a dashboard. The Card is visually identical to the [Ignite UI for Angular Card Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/card.html)
+
+## Card Demo
+
+<img class="responsive-img" src="../images/card_demo.png" srcset="../images/card_demo@2x.png 2x" />
+
+## Areas
+
+The Card has four distinct areas: for `image` at the top to show picture or map media, a `header` which holds a combination of thumbnail Avatar, title, and subtitle, `body` which holds text paragraphs detailing the Card, and `actions` which serves as a toolbar with quick actions related to the Card.
```
Comment (sdimchevski) [inline en/components/card.md:17]:
```suggestion
The Card has four distinct areas: `image` at the top to show picture or map media, `header` which holds a combination of thumbnail Avatar, title, and subtitle, `body` which holds text paragraphs detailing the Card, and `actions` which serve as a toolbar with quick actions related to the Card.
```

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**